### PR TITLE
Create html5-viewer-styles-broken-with-kendo-2022-1-119.md

### PR DESCRIPTION
**Pull request by todorarabadzhiev**

### CODE REVIEW

File: knowledge-base/html5-viewer-styles-broken-with-kendo-2022-1-119.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ticketid: 1551073
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product Version</td>
+			<td></td>
```
Comment (DimitarNikolovv06) [inline knowledge-base/html5-viewer-styles-broken-with-kendo-2022-1-119.md:18]: The version is missing.

File: knowledge-base/html5-viewer-styles-broken-with-kendo-2022-1-119.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+		</tr>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
```
Comment (DimitarNikolovv06) [inline knowledge-base/html5-viewer-styles-broken-with-kendo-2022-1-119.md:23]: You may add a row that this is for the HTML5 report viewers

File: knowledge-base/html5-viewer-styles-broken-with-kendo-2022-1-119.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+	</tbody>
+</table>
+
+
+## Description
+There are styling issues in the [Html5 Report Viewer](h../using-reports-in-applications/display-reports-in-applications/web-application/html5-report-viewer/overview) and its wrappers when
```
Comment (DimitarNikolovv06) [inline knowledge-base/html5-viewer-styles-broken-with-kendo-2022-1-119.md:29]: the link starts with h

File: knowledge-base/html5-viewer-styles-broken-with-kendo-2022-1-119.md (hunk 1 above)
Comment (todorarabadzhiev) [reply knowledge-base/html5-viewer-styles-broken-with-kendo-2022-1-119.md:18]: It is relevant to all Reporting versions that try to use this specific Kendo version. That's why the Reporting version is not included.

**Review by DimitarNikolovv06 — APPROVED** [review]

**Pull request merged**